src/Codeception/Exception/UselessTestException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Codeception\Exception;
+
+use PHPUnit\Framework\AssertionFailedError;
+
+class UselessTestException extends AssertionFailedError
```

Comment:
Why didn't you call it `UselessTestError` if it extends from `AssertionFailedError`?

Reply:
Because it is Codeception exception and we don't have any `Error` classes.